src/renderer/atlas/TextureLoader.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+namespace Microsoft::Console::Render
+{
+    struct ShaderTexture
+    {
+        wil::com_ptr<ID3D11Resource> Texture;
+        wil::com_ptr<ID3D11ShaderResourceView> TextureView;
+    };
+
+    Microsoft::Console::Render::ShaderTexture LoadShaderTextureFromFile(

```

Comment:
I put this in a separate file as I felt that could help if wanted to add the function to DxEngine as well? What is the status of DxEngine? How do I test it?

Reply:
Our intention is to slowly replace DxEngine with AtlasEngine over time. As such we don't really plan to implement any improvements for it anymore unless they have a great usability impact for everyone. Personally, I wouldn't port this PR over to DxEngine.

Reply:
Ok

Reply:
I might leave it as is as I don't think it hurts terribly having it in its own file and might help a unit test

Reply:
I agree with your intuition here @mrange -- I don't think DxEngine is dead yet, especially given that we have some issues with NewNewAtlasEngine that may prevent broad adoption . . . today. :smile: